CHANGES.rst

```diff
@@ -4,13 +4,22 @@ Changes
 Unreleased
 ----------
 
-- Fixed ``prev_record`` and ``next_record`` performance when using ``excluded_fields`` (gh-791)
+Upgrade Implications:
+
+- Run `makemigrations` after upgrading to realize the benefit of indexing changes.
+
+Full list of changes:
+
+- Added index on `history_date` column; opt-out with setting `SIMPLE_HISTORY_DATE_INDEX` (gh-565)
+- Added `included_fields` for `history.diff_against` (gh-776)
+- Added Czech translations (gh-885)
```

Comment:
Is this correct?

Reply:
Yes it is correct.  I merged a couple PRs that did not have entries in CHANGES.rst so I added them here.  I will clean this up so you can approve it.